post a comment added

Antony-Lester opened the pull request.

Commit:
post a comment added

Antony-Lester commented:
9. Post a new comment to an existing article

Commit:
Update API.js

removed console.log

Augs0 reviewed (changes requested):
Great effort so far! There do seem to be some issues with the functionality though. When I try to post I get the following error:

`Cannot read properties of undefined (reading 'preventDefault')`

After this, I was unable to get the post comment box to appear on any other articles, even after refresh. Could you look into this issue?

I would also suggest perhaps thinking about the user experience. It wasn't immediately clear to me what the button with the person reading did i.e. that it would show and hide comments

Commit:
removed images on buttons and correced code

Commit:
console.log removed
Merge branch '8-comments' of https://github.com/Antony-Lester/frontend-news into 8-comments

Antony-Lester commented:
note:  it will only allow a user to add a comment if they have not commented on the article already, I plan to make this clearer to the user on task 11,

Commit:
Update article.jsx

removed a comment log
